tests/test_prepared.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+pytestmark = pytest.mark.asyncio
+
+
+class TestPreparedStatement:
+
+    async def test_bind(self, session):
+        statement_str = "INSERT INTO test (id, value) values( ?, ?)"
+        prepared = await session.create_prepared(statement_str)
+        statement = prepared.bind()
```

Comment:
shouldn't this fail ? i.e. how can you bind with no parameters ?
would be nice if this can get list or dict

Reply:
Behind the scenes, it basically does the `cass_prepared_bind` [1] which is different from binding a variable to final statement.

Regarding how easy or difficult is make variable bindings as it is done right now my plan would be:

- First, have a solid foundation API similar to the C++ API, this API should be usable in production despite the verbosity that it might imply.
- Implement a thin Python wrapper on top of that which would be exposed as a kinda ORM. This would be done in the following release, this should remove the verbosity of the raw API.

[1] https://docs.datastax.com/en/developer/cpp-driver/2.3/api/struct.CassPrepared/#function-cass_prepared_bind